daktari/check_runner.py

```diff
@@ -56,11 +52,14 @@ def run_check_in_try(self, check: Check) -> CheckResult:
             logging.debug(f"Exception running check {check.name}", exc_info=True)
             return CheckResult(check.name, CheckStatus.ERROR, f"Check failed with unhandled {type(err).__name__}", {})
 
-    def diagnose_missing_dependency(self, check: Check):
+    def diagnose_missing_dependency(self, check: Check) -> CheckResult:
         all_checks = {check.name for check in self.checks}
         check_dependencies = {dependency.name for dependency in check.depends_on}
         missing_checks = check_dependencies.difference(all_checks)
-        if check_dependencies.difference(all_checks):
-            print(f"⚠️  [{yellow(check.name)}] skipped due to missing dependent checks: {', '.join(missing_checks)}")
-        else:
-            print(f"⚠️  [{yellow(check.name)}] skipped due to previous failures")
```

Comment:
Deliberate choice to drop the yellow?

Reply:
Ah no I see, we now get a check result with `CheckStatus.PASS_WITH_WARNING` so `print_check_result` will print the name in yellow :+1: